torch_glow/tests/utils.py

```diff
@@ -70,4 +70,4 @@ def jitVsGlow_(f_torch, f_glow, *inputs, expected_fused_ops=None,
         # different, some ops in expected_fused_ops are not in the graph at all
         assert accept_all_ops or len(expected_fused_ops) == len(expected_fused_ops_seen), \
             "Expected all of expected_fused_ops to be in the graph"
-        assert torch.allclose(torch_res, glow_res, atol=01e-6)
+        assert torch.allclose(torch_res[0], glow_res[0], atol=01e-6)
```

Comment:
For anything other than tuples this will just take a slice which isn't what we want
Let's make our own allClose function that does the correct thing for tuples

Reply:
For tuples of tensors, lets just all torch.allclose on all elems